src/acb_matrix.jl

```diff
@@ -0,0 +1,50 @@
+function AcbMatrix(A::AbstractMatrix{Acb}; prec::Integer = precision(first(A)))
+    M = AcbMatrix(size(A)...; prec = prec)
+    for j = 1:size(A, 2), i = 1:size(A, 1)
+        M[i, j] = A[i, j]
+    end
+    M
+end
+
+Base.size(M::AcbMatrix) = (M.acb_mat.r, M.acb_mat.c)
+Base.precision(M::AcbMatrix) = M.prec
```

Comment:
I would move this to `precision.jl`, also add `AcbMatrix` and `acb_mat_struct` to some of the other methods defined there.